test/markdownlint-test.js

```diff
@@ -1032,6 +1032,44 @@ tape("configMultiple", (test) => {
     });
 });
 
+tape("configMultipleWithRequireResolveBefore", (test) => {
+  test.plan(0);
+  try {
+    fs.symlinkSync(
+      path.resolve(__dirname, "./config/pseudo-package"),
+      path.resolve(__dirname, "../node_modules/pseudo-package"),
```

Comment:
Could we place this into a `node_modules` under the `test` directory instead? That will keep it out of the "real" one and it should still be searched first, I think?

Reply:
Although there exists a [`test`](https://www.npmjs.com/package/test) package on npm, I renamed as you suggested. I don't think this will ever cause a clash.

Reply:
Symlink creation should work on Windows too. CI does not seem to complain.